### PR TITLE
Get start development without npm

### PR DESCRIPTION
**Pull request by shunsuke-excite**
## What type of PR is this? (check all applicable)

- [x] Other

## Description
*  docker-compose can `up`  without  installing `npm`  
* `Postgres`  can save persistently

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ services:
       - "5678:5678"
     volumes:
       - ".:/app"
+      - "/app/client/dist"
```
Comment (arikfr) [inline docker-compose.yml]: Won't this break using npm on the host?

Comment (shunsuke-excite) [reply docker-compose.yml]: With using `npm` for development on host,  `npm start` on host for development.

Comment (arikfr) [reply docker-compose.yml]: I meant, wouldn't setting up the volume this way make the changes on the host not seen in the container?

Comment (shunsuke-excite) [reply docker-compose.yml]: I see. 
I'll update the changes made on the host can appear  in the container.

Comment (shunsuke-excite) [reply docker-compose.yml]: I'm sorry, I misunderstood.
This Pull Request close.

**Pull request closed without merging**